app/components/learner-select-field.js

```diff
@@ -0,0 +1,6 @@
+import UISelectFieldComponent from 'ui-select/components/ui-select-field';
+import layout from '../templates/components/learner-select-field';
+
+export default UISelectFieldComponent.extend({
+  layout
```

Comment:
I don't understand how this line works.  Shouldn't it need to be `layout: layout`?